src/main/java/co/omise/models/Transaction.java

```diff
@@ -76,7 +85,7 @@ public GetRequestBuilder(String transactionId) {
         }
 
         @Override
-        protected HttpUrl path() throws IOException {
+        protected HttpUrl path() {
```

Comment:
Why do we need to delete the `throws` from method signature?

Reply:
They are unneeded, from what I could see. Also got a warning in the IDE.